using/submodules

```diff
@@ -0,0 +1,48 @@
+# Use Git submodules
```

Comment:
According to other articles, it should be "Using Git submodules" (same as in the summary)